aws-serverless-java-container-core/src/test/java/com/amazonaws/serverless/proxy/internal/servlet/AwsAsyncContextTest.java

```diff
@@ -58,6 +73,7 @@ void dispatch_sendsToCorrectServlet() {
         assertEquals(202, handler.getResponse().getStatus());
     }
 
+    @Disabled("AwsAsyncContext does not sends to servlet anymore")
```

Comment:
In my view we should remove the two disabled test cases as nobody will re-enable them in the future.

Reply:
Removed `AwsLambdaServletContainerHandler` from `AwsAsyncContext` and these two Disabled tests.